addons/mail/static/src/composer/composer.js

```diff
@@ -409,11 +411,40 @@ export class Composer extends Component {
         markEventHandled(ev, "composer.clickOnAddEmoji");
     }
 
+    onClickAddGif() {
```

Comment:
could just be `onClickGif` / `onClickGifButton`, since it actually a toggle func and not only for add

Reply:
This is to be similar to `onClickAddEmoji`. I don't think it's a big deal.